src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        return String.valueOf(Colors.values()[new Random().nextInt(Colors.values().length)]);
```

Comment:
Using 'Colors.values()' inside the loop causes it to create a new array every time 'getRandomColor()' is called, which is not efficient. Store the result of 'Colors.values()' in a constant field and use that field here.